### DISCUSSION TITLE
NAN values in loss, perplexity and next_token_perplexity

### DISCUSSION THREAD

**Discussion by msmmpts** [ludwig-ai/ludwig#3781] [Q&A]
Hi team,

I was attempting to fine tune a Japanese LLM for a domain specific task. Training completed without any issues but the logs show NAN values for loss, perplexity, ..etc. I am uploading description.json and training_statistics.json for review.

[description.json](https://github.com/ludwig-ai/ludwig/files/13374368/description.json)
[training_statistics.json](https://github.com/ludwig-ai/ludwig/files/13374370/training_statistics.json)

Any help regarding the same would be really appreciated.

Thanks in advance

**Comment by arnavgarg1** [comment]
Hi @msmmpts, sorry for the delayed response on this discussion thread. Is this still an issue you're running into with the latest stable release of Ludwig (0.9.3)? Happy to help debug!
